services/data-warehouse/app/main.py

```diff
@@ -6,9 +6,16 @@
 from app.logs.extract import logs_workflow, logs_task
 from app.events.extract import events_workflow, events_task
 
+# Import materialized views
+from app.views.daily_pageviews import daily_pageviews_mv
+
+# Export materialized views (similar to TypeScript pattern)
+__all__ = ['daily_pageviews_mv']
```

Comment:
Hm, do we need this?

Reply:
No we don't.  I removed it in the next PR update.